docs/_config.yml

```diff
@@ -0,0 +1,2 @@
+theme: jekyll-theme-cayman
+google_analytics: [UA-126581035-2]
```

Comment:
That's my personal code.

Reply:
Please double check if it counts the views. And I guess Github pages is not yet activated.